server/boot/observe-changes.js

```diff
@@ -0,0 +1,42 @@
+var _ = require('lodash');
+var debug = require('debug')('strong-mesh-models:server:observe-changes');
+
+module.exports = function observeChanges(server) {
+  var ServiceInstance = server.models.ServiceInstance;
+  var ServiceProcess = server.models.ServiceProcess;
+  var ServerService = server.models.ServerService;
+
+  ServiceInstance.observe('after save', function(ctx, next) {
+    var inst = ctx.instance || ctx.data;
+    ServerService.__changeEmitter.emit('svc-' + inst.serverServiceId, {
+      target: 'ServiceInstance-' + inst.id,
+      type: ctx.isNewInstance ? 'create' : 'update',
+      data: _.omit(inst.toObject(), [
```

Comment:
Would it be better to take a whitelist approach here instead and list only the fields that we _do_ want? That way in the future if a sensitive field is added to the model we can't accidentally forget to hide it here.